spec/abstractops.html

```diff
@@ -1442,4 +1442,190 @@ <h1>ParseTemporalYearMonthString ( _isoString_ )</h1>
         }.
     </emu-alg>
   </emu-clause>
+
+  <emu-clause id="sec-temporal-topositiveintegerorinfinity" aoid="ToPositiveIntegerOrInfinity">
+    <h1>ToPositiveIntegerOrInfinity ( _argument_ )</h1>
+    <p>
+      The abstract operation ToPositiveIntegerOrInfinity takes argument _argument_. It converts _argument_ to +&infin; or an integer _i_ for which <emu-eqn>0 &lt; i</emu-eqn>. It performs the following steps:
+    </p>
+    <emu-alg>
+      1. Let _integer_ be ? ToInteger(_argument_).
+      1. If _integer_ is *-&infin;*<sub>𝔽</sub>, then
+        1. Throw a *RangeError* exception.
+      1. If _integer_ &le; 0, then
```

Comment:
```suggestion
      1. If _integer_ ≤ 0, then
```

Reply:
(will treat the same as &infin;, whichever way it goes)